client/src/helpers/APICalls/stripe.ts

```diff
@@ -13,4 +13,14 @@ const getLink = async (): Promise<StripeLink> => {
     }));
 };
 
-export default getLink;
+export const newCust = async (): Promise<string> => {
```

Comment:
Let's make this `newCustomer`. More descriptive 🙂

Reply:
Got it - thanks! Ended up removing this API call now that logic is in the back-end.